network/address.go

```diff
@@ -13,7 +13,8 @@ import (
 func (c *Client) GetDataNetworkIP() (net.IP, error) {
 	re := c.runenv
 	if !re.TestSidecar {
-		return nil, ErrNoTrafficShaping
+		// this is a local runner; return the loopback address.
```

Comment:
```suggestion
		// this must be a local:exec runner and we currently don't support traffic shaping on it
		// for now, just return the loopback address
```

I suggest a slightly more verbose comment as this is really a hack to do some kind of `data network` no-op.

Reply:
sure!